### PR TITLE
Docs: Updates for vault-mssql-ekm-provider-enterprise release v0.2.1

### PR DESCRIPTION
**Pull request by tomhjp**
v0.2.1 available [here](https://releases.hashicorp.com/vault-mssql-ekm-provider/0.2.1+ent/).

## 0.2.1 (June 30th, 2023)

BUGS

* The provider now correctly uses a previous key version for decryption when the transit key is rotated but the DEK has not yet been re-encrypted.
* Fixed a bug that prevented the provider from emitting Event logs.

### CODE REVIEW

**Review by thyton — APPROVED** [review]

File: website/content/docs/platform/mssql/troubleshooting.mdx [hunk 1]
```diff
@@ -19,11 +19,37 @@ your issue, you can [enable trace logging](/vault/docs/platform/mssql/configurat
 Restart SQL Server for the config change to take effect, and you should see more
 detailed logs in the same section of Windows Event Viewer.
 
+## Check SQL Server error logs
+
+If the Vault EKM provider is not generating event logs, you may find some
+information in the SQL Server's error logs. Check for an event with ID
+17111 from "MSSQLSERVER" which details the file location of the error logs,
+e.g. "Logging SQL Server messages in file
```
Comment (tomcf-hcp) [inline website/content/docs/platform/mssql/troubleshooting.mdx:27]: `for example` instead of `e.g.`?

**Review by benashz — APPROVED** [review]
LGTM!

File: website/content/docs/platform/mssql/troubleshooting.mdx [hunk 2]
```diff
@@ -19,11 +19,37 @@ your issue, you can [enable trace logging](/vault/docs/platform/mssql/configurat
...
+DLL" in the error logs. This can normally be remediated by updating the CA trust
+store to include the latest Digicert certificates. Run the following to import
+the required certificates into the Root trust store:
+
+```powershell
+New-Item -ItemType Directory -Path certs
+certutil -syncwithWU .\certs\
+extrac32 -L .\certs\ .\certs\authrootstl.cab
+certutil -f -ent -AddStore Root .\certs\authroot.stl
+certutil -f -ent -AddStore Root .\certs\0563b8630d62d75abbc8ab1e4bdfb5a899b24d43.crt
```
Comment (tomcf-hcp) [inline website/content/docs/platform/mssql/troubleshooting.mdx:43]: I assume these are example cert names. If so, we should add a note before "Run the following..." to also replace the cert names with actual values.

**Review by tomcf-hcp — APPROVED** [review]

File: website/content/docs/platform/mssql/troubleshooting.mdx (hunk 2 above)
Comment (tomhjp) [reply website/content/docs/platform/mssql/troubleshooting.mdx:43]: They're actually the exact Digitcert ones required :) the -syncwithWU command above downloads dozens of certs.

**Pull request merged**